hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OFSPath.java

```diff
@@ -163,6 +163,14 @@ public String getKeyName() {
     return keyName;
   }
 
+  /**
+   * Returns the last element of the KeyName, which can either be a file name
+   * or the leaf directory/bucket.
+   */
+  public String getFileName() {
```

Comment:
This can be a  directory too  i.e `mv vol1/buck1/dir1/dir2 vol1/buck1/`

Reply:
yes, you are right, that's why I documented that in the javadoc. let me know if you have a better name idea for the method :)

Reply:
Maybe getLastElement()?